### DISCUSSION TITLE
Override text: "Push It"

### DISCUSSION THREAD

**Discussion by nanos** [pglombardo/PasswordPusher#1361] [Q&A] [answered]
Hi, 

is it possible to override text strings somehow? 

E.g. I'd like to rename 'Push it' to 'Share' in the English version.

I tried to mount a modified version of `config/locales/gettext/en/app.po` like this `--mount type=bind,source=/path/to/en-app.po,target=/opt/PasswordPusher/config/locales/gettext/en/app.po` but that doesn't seem to have any effect?

Thanks!

**Comment by pglombardo** [comment, marked as answer]
Unfortunately, it's not currently possible to override that string.

Overriding the `.po` file is a good idea but probably doesn't work since English is the source language.

The line of code with that string [is here](https://github.com/pglombardo/PasswordPusher/blob/master/app/views/passwords/new.html.erb#L181).

**Reply by nanos** [reply]
OK, thanks!

**Comment by anikwai** [comment]
@pglombardo I have the same question, Is it still not possible?

**Reply by pglombardo** [reply]
Not yet but in progress slowly as I migrate strings to a centralized external file...

**Comment by atimberlake** [comment]
Same here, as many users think clicking the button pushes to password to the correct team instead of creating a link. Would love a variable for this.

**Reply by atimberlake** [reply]
Curious if this was completed during the consolidation @pglombardo. Thank you for all your work on this.

**Reply by pglombardo** [reply]
Hi @atimberlake - I've had a couple of reports of this being confusing.  I have this on the list to do but in the meantime, a hacky fix would be to modify [line 985 of this file](https://github.com/pglombardo/PasswordPusher/blob/master/config/locales/gettext/en/app.po#L985).  You would have to keep a local copy of that file and then setup a [Docker file overlay](https://docs.pwpush.com/docs/overlay-file-docker/).  It's ugly but it would work for now.
